pg_search/src/postgres/customscan/pdbscan/projections/snippet.rs

```diff
@@ -26,6 +26,7 @@ use std::ptr::addr_of_mut;
 
 const DEFAULT_SNIPPET_PREFIX: &str = "<b>";
 const DEFAULT_SNIPPET_POSTFIX: &str = "</b>";
+const DEFAULT_SNIPPET_MAX_NUM_CHARS: i32 = 150;
```

Comment:
Is this a hardcoded limit of Tantivy? How did you get to 150? If so, we should document it.

Reply:
It just follows the previous definition.

https://github.com/paradedb/paradedb/blob/bb12d9752012eb8bd93180048b0d0731a4ab53e3/pg_search/src/postgres/customscan/pdbscan/projections/snippet.rs#L39-L47

Reply:
@rebasedming @neilyio any clue?

Reply:
> Sets a maximum number of chars. Default is 150.

https://docs.rs/tantivy/latest/tantivy/snippet/struct.SnippetGenerator.html#method.set_max_num_chars

Reply:
Ah alright and this is already documented. Sorry for the silly questions, and thank you for taking this issue.